recipe/meta.yaml

```diff
@@ -37,28 +37,29 @@ requirements:
     - {{ posix }}zip                 # [win]
   host:
     - r-base
-    - libcurl                        # [not win]
+    - libcurl
   run:
     - r-base
-    - {{ native }}gcc-libs           # [win]
-    - {{ native }}libwinpthread-git  # [win]
-    - {{ native }}zlib               # [win]
 
 test:
   requires:
+{% if version != "5.1.0" or build != 1 %}  # temporarily disabled for migration
     - r-httpuv
     - r-testthat
     # NB: r-webutils creates a circular dependency; it should be temporarily disabled during migrations
     - r-webutils
+{% endif %}
   source_files:
     - tests/
     - tests/testthat
   commands:
-    - $R -e "library('curl')"                                                                 # [not win]
+    - $R -e "library('curl')"           # [not win]
+    - "\"%R%\" -e \"library('curl')\""  # [win]
+{% if version != "5.1.0" or build != 1 %}  # temporarily disabled for migration
     - $R -e "testthat::test_file('tests/testthat.R', stop_on_failure=TRUE)"                   # [not win]
     - $R -e "invisible(curl::curl_fetch_memory('http://rest.kegg.jp/conv/ncbi-geneid/dme'))"  # [not win]
```

Comment:
FYI, I'd expect this to fail when this is built against a recent `libcurl` that requires `openssl>=3.2`.
See https://github.com/openssl/openssl/issues/11378 via https://github.com/curl/curl/issues/13389 .